src/Socialengine/SnifferRules/Command/SniffCommand.php

```diff
@@ -232,4 +136,69 @@ protected function getOptions()
     {
         return array();
     }
+
+    protected function processOptions()
+    {
+        $standards = $this->config->get('sniffer-rules.standard', array('PSR2'));
+        $files = $this->config->get('sniffer-rules.files', array('app/models', 'app/controllers'));
```

Comment:
no, sorry, `app/models` and `app/controllers` [no longer](https://mattstauffer.co/blog/laravel-5.0-directory-structure-and-namespace) exist in laravel 5. I believe its safe to just do `app/`